README.md

```diff
@@ -168,102 +159,76 @@ Aleth supports EVMC out of the box, but geth not yet.
+	--vmodule "eth=12,p2p=12" \
+	--nodiscover
 	```
 
-	*NOTE*: don't forget to specify `networkId` with the same value as the value of `chainID` in the genesis configuration, this is to avoid [Metamask error `Invalid Sender`](https://github.com/MetaMask/metamask-extension/issues/3673).
-
 	Note that if you want your node to be automatically restarted if it dies, and to survive system reboots, you'll want to use a tool such as [pm2](http://pm2.keymetrics.io/):
 
 	```sh
 	> npm install -g pm2
 	```
 
+### geth + Wagon
 
-## geth + Wagon
+The testnet has experimental support for running a geth node with the [Wagon](http://github.com/go-interpreter/wagon) third-party VM. Instructions may be found [here](wagon.md).
+
+### Aleth (cpp-ethereum) + Hera
 
 **NOTE: this client currently is not supported fully and the instructions here may be wrong.**
 
-This section describes how to run geth node with the [Wagon](http://github.com/go-interpreter/wagon) VM.
+Support for aleth (formerly, cpp-ethereum) is a work in progress and more information may be found [here](aleth.md).
 
-1. Get the code from the PR
 
-	```sh
-	> go get github.com/ethereum/go-ethereum
-	> cd $GOROOT/src/github.com/ethereum/go-ethereum
-	> git remote add gballet git@github.com:gballet/go-ethereum.git
-	> git fetch gballet add-ewasm
-	> git checkout add-ewasm
-	```
+## Syncing to the testnet
+
+In order to sync to the testnet, you'll need the proper genesis configuration.
 
-2. Build geth
+1. Follow the steps above to download and build geth+Hera.
+
+1. Download the [genesis file](ewasm-testnet-geth-config.json) and use it to initialize geth:
 
 	```sh
-	> go build ./cmd/geth/...
+	> wget https://raw.githubusercontent.com/ewasm/testnet/ewasm-testnet-milestone1/ewasm-testnet-geth-config.json
+	> ./build/bin/geth --datadir /tmp/ewasm-node/4201/ init ewasm-testnet-geth-config.json
 	```
 
-3. Run geth
+	Note that the `/tmp/ewasm-node/4201` directory name above is arbitrary. It just needs to be unique.
+	
+	Note also the `--etherbase`, `--networkid`, and `--bootnodes` commands, below, and copy them verbatim as these are required to connect to and sync with the Ewasm testnet.
 
-	The ewasm interpreter will be activated by default as long as you do not specify a `--vm.ewasm=...` on the command line option.
+	Don't forget to specify `networkId` with the same value as the value of `chainID` in the genesis configuration to avoid [Metamask error `Invalid Sender`](https://github.com/MetaMask/metamask-extension/issues/3673).
+	
+	Run geth as above, but add the following commandline args:
 
 	```sh
-	> ./geth \
-	--datadir /tmp/ewasm-node/4201/ \
-	--etherbase 031159dF845ADe415202e6DA299223cb640B9DB0 \
-	--rpc --rpcapi "web3,net,eth,debug" \
-	--rpcvhosts="*" --rpcaddr "0.0.0.0" \
-	--rpccorsdomain "*" \
-	--mine --miner.threads 1 \
-	--nodiscover \
+	--etherbase a8c3eeb2915373139bcfc287d4ae9e660d734881 \
 	--networkid 66 \
```

Comment:
Why is it better to split these two instructions? It seems weird that this block only contains extra options.

Reply:
Are you referring to the `geth` configuration specified at [line 174](https://github.com/ewasm/testnet/pull/102/files#diff-04c6e90faac2675aa89e2176d2eec7d8R174)? Because there's a difference between running a node locally and running a node that syncs the testnet -- these additional commands are intended for testnet syncing. (Though I think `--etherbase` could be removed)

Reply:
What is the use of "running a node locally" without connecting to the testnet?

FYI running the node locally can be done in "test" mode which avoids 90% of the commandline options.

Reply:
> What is the use of "running a node locally" without connecting to the testnet?

Local development.

> FYI running the node locally can be done in "test" mode which avoids 90% of the commandline options.

Okay. Presumably we still need the `--vm.ewasm` flag but not the RPC-related ones?